packages/calcite-components/src/components/tooltip/tooltip.tsx

```diff
@@ -344,7 +344,7 @@ export class Tooltip implements FloatingUIComponent, OpenCloseComponent {
         >
           <FloatingArrow
             floatingLayout={floatingLayout}
-            ref={(arrowEl: SVGElement) => (this.arrowEl = arrowEl)}
+            ref={(arrowEl) => (this.arrowEl = arrowEl)}
```

Comment:
no longer need to type this here now that FloatingArrow ref type is typed correctly